### DISCUSSION TITLE
Lower data usage

### DISCUSSION THREAD

**Discussion by Welsyntoffie** [python-telegram-bot/python-telegram-bot#2641] [Q&A]
It is not required for my purpose for the script to poll the server at high speeds. It uses extra metered data.
What parameters needs to be changed to check maybe every 10 seconds. I am aiming to reduce the data usage by about 80%.

**Comment by Bibo-Joshi** [comment]
`Updater.start_polling` has a parameter called `poll_interval`. However, I don't know enough about memory usage & networking to tell if that actually reduces memory usage. You also might want to look into webhooks.

**Reply by Welsyntoffie** [reply]
I did look into webhooks, but is much more complicated setup...
```
You have to obtain an URL
You have to obtain and set up an SSL certificate for the URL
You have to set up a web server to handle the POST requests coming from Telegram servers
```
Just to be more clear. It is not an issue with memory usage. I have a metered internet connection. At the current rate it uses +-114 megabytes of internet data per day, around 4.5 megabytes internet per hour.

**Reply by harshil21** [reply]
You could just host your bot on some platform like Heroku. See https://github.com/python-telegram-bot/python-telegram-bot/wiki/Where-to-host-Telegram-Bots

**Reply by Welsyntoffie** [reply]
The PC ethernet port is connected to a network of PLC's. I use telegram to get status of motors and pumps. Thanks anyway.
My raspberry pi does not use that much data using the same nethogs monitor. Maybe its a platform thing. buster vs Ubuntu 20

**Reply by Poolitzer** [reply]
I guess Bibo read the memory part wrongly, increasing the poll interval should be exactly what you want.
